src/main/java/com/patreon/resources/shared/SocialConnections.java

```diff
@@ -43,6 +43,7 @@ public UserIdObject getSpotify() {
   public static class UserIdObject {
     private String user_id;
     private List<String> scopes;
+    private String url;
```

Comment:
@emosesPatreon The `url` comes from the objectMapper? Is objectMapper able to change private variables here? Seems weird

Reply:
Yeah, that's how Jackson deserialization works (well, one of the ways it can work)